cmd/wit-bindgen-go/main_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"testing"
+)
+
+const simpleHttpWitPath = "../../testdata/codegen/simple-http.wit"
```

Comment:
No need to have a const here that's only used once.

Reply:
Will update this!